Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	
+++ b/build.gradle	
@@ -25,7 +25,7 @@
 
 application {
     mainModule = 'com.team1.ce216project'
-    mainClass = 'com.team1.ce216project.HelloApplication'
+    mainClass = 'com.team1.ce216project.Main'
 }
 
 javafx {
